alerts/ui/docker/README.md

```diff
@@ -0,0 +1,37 @@
+# Supernova Docker Image
+
+This Dockerfile builds a standalone, runnable Supernova image. You can configure application properties using environment variables.
+
+## Build the Image
+
+Ensure you are in the `alerts/ui` directory, then build the Docker image with:
+
+```bash
+docker build -t supernova -f docker/Dockerfile .
+```
+
+## Run the Container
+
+```bash
+docker run -e ENDPOINT="https://alertmanager_url/api/v2" -e THEME="theme-light" -p 3010:80 supernova
```

Comment:
Remove passing of theme env from the default run command. We can add a second example that shows how env variables can be passed:
```suggestion
docker run -e ENDPOINT="https://alertmanager_url/api/v2" -p 3010:80 supernova
```

Reply:
Can the env variables also be passed from a file? The more complex ones like silence templates will be hard to do in a CLI command